packages/esm-patient-chart-app/src/workspace/workspace-window.component.tsx

```diff
@@ -89,7 +89,10 @@ const WorkspaceWindow: React.FC<ContextWorkspaceParams> = () => {
                 <HeaderGlobalAction
                   align="bottom-right"
                   label={t('close', 'Close')}
-                  onClick={() => closeWorkspace?.()}
+                  onClick={() => {
+                    closeWorkspace?.();
+                    launchPatientWorkspace('order-basket');
+                  }}
```

Comment:
This is the workspace-window and this will take place for every workspace.

We need to set an `onCloseWorkspace` which should be called after the `drug-search-workspace` is closed.